.eslintrc

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
This PR adds ESLint rules that I am not comfortable with. This is something we are discussing in the new version of WPGulp. You're more than welcomed to discuss it at WPGulp 2.0 Call For Contributors here → #61